ledger/store/trackerdb/sqlitedriver/sql.go

```diff
@@ -421,6 +426,86 @@ func (qs *accountsDbQueries) LookupAllResources(addr basics.Address) (data []tra
 	return
 }
 
+func (qs *accountsDbQueries) LookupLimitedResources(addr basics.Address, minIdx basics.CreatableIndex, maxCreatables uint64, ctype basics.CreatableType) (data []trackerdb.PersistedResourcesDataWithCreator, rnd basics.Round, err error) {
```

Comment:
I believe the `ctype` argument is no longer used

Reply:
This reminded me of problem I have - if creator is an optional/left join, then there is no reasonable way to filter for assets in the query for otherwise "deleted" resources. I'm not necessarily sure what the best solution to this is without adding creatable type to the resources table.

Reply:
Addressed with migration.